src/codegen/browser/types.ts

```diff
@@ -98,10 +98,16 @@ export interface IsCheckedAssertion {
   expected: CheckState
 }
 
+export interface HasValuesAssertion {
+  type: 'has-values'
+  expected: [string, ...string[]]
+}
```

Comment:
I don't make any distinction whether we're asserting a single value or multiple values here. I instead opted to select which method to use (`toHaveValue` or `toHaveValues`) when emitting the code. Branching here would just mean more code to achieve the same thing.